src/lib/components/explorer/cell/Actions.svelte

```diff
@@ -1,11 +1,10 @@
 <script lang="ts">
   import type { SearchResult } from '$lib/models/Search';
   import { activeTable, expandableComponents, setActiveRow } from '$lib/stores/ExpandableRow';
-  import type { Export } from '$lib/models/Export';
   import ExportStore from '$lib/stores/Export';
   let { exports, addExport, removeExport } = ExportStore;
   export let data = {} as SearchResult;
-  let exported: Export = {
+  $: exported = {
     variableName: data.row.name,
     variableId: data.row.id,
   };
```

Comment:
I think you can keep type safety with something like:
```suggestion
  } as Export;
```
Can you check if that works?

Reply:
I updated this. I also noticed a bug where it was no longer identifying the Export object as the same object for array includes purposes.